app/controllers/profiles_controller.rb

```diff
@@ -1,5 +1,47 @@
 class ProfilesController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @profile = Profile.new
+  end
+
+  def create
+    @profile = Profile.new(profile_params)
+    @user = User.find(params[:user_id])
+    @profile.save
```

Comment:
Use an If Else statement for saving a created instance. We need to show an error message if it does not save.